sql/core/src/main/scala/org/apache/spark/sql/util/QueryExecutionListener.scala

```diff
@@ -109,6 +109,16 @@ class ExecutionListenerManager private[sql](session: SparkSession, loadExtension
     listenerBus.removeAllListeners()
   }
 
+  /**
+   * Removes [[ExecutionListenerBus]] from LiveListenerBus.queues. people should call this function
```

Comment:
Need rewording IMHO.

"This function should be called..." (no idea what should actually follow. it's not clear to me by the change)

Reply:
`SparkSession.clone()` will create a new `ExecutionListenerBus` and each `ExecutionListenerBus` reference its belongs session, before this change we have no method to remove `ExecutionListenerBus`, if we continuously cloning session from our application, these session/listeners will occur a memory leak. So we should clear `ExecutionListenerBus` through this method when we remove the cloned session.

Reply:
hi, @jaceklaskowski
what do you think of the comment shows as follows?
> Removes [[ExecutionListenerBus]] from LiveListenerBus.queues.
> Each SparkSession has its owner listener manager, and this manager won't clear by Spark. when you destroy a session, you should call this function to clean up the listener manager.

Reply:
Better. What do you think of this?

> Removes [[ExecutionListenerBus]] from LiveListenerBus.queues.
> Each SparkSession has its own listener manager, and this manager won't be cleared by Spark.
> Developers are expected to call this function (to clean up the listener manager) while destroying a session

What do you think about a code snippet to help devs understand it even better? Code like picture says a thousand words :)

Reply:
updated.

Reply:
looks fine.